committing.rst

```diff
@@ -235,10 +251,10 @@ it is common to simply paste the NEWS entry into the commit message.  Here
 is an example::
 
    bpo-42: the spam module is now more spammy.
```

Comment:
This is part of reST markup for a block literal so it actually doesn't when viewed as rendered HTML.